.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,42 @@
+name: Run Gradle tests
+
+on: push
+
+env:
+  artifact_path: /tmp/external-system-test-api.zip
+  PANTS_SHA: "1.25.x-twtr"
```

Comment:
Personally I'd prefer to have git SHA instead of branch head here

Reply:
This was taken from `travis.yml`, changed to SHA